src/scanner.js

```diff
@@ -45,6 +45,54 @@ module.exports = function ({ config, db, logger }) {
     })
     .subscribe()
 
+  // Clean db
+  async function cleanDeleted () {
+    logger.info('Checking for dead media')
+    const deleted = []
+
+    const limit = 256
+    let startkey = undefined
+    while (true) {
+      const { rows } = await db.allDocs({
+        include_docs: true,
+        startkey,
+        limit
+      })
+      await Promise.all(rows.map(async ({ doc }) => {
+        try {
+          if (doc.mediaPath.indexOf(config.scanner.paths) === 0) {
+            try {
+              const stat = await statAsync(doc.mediaPath)
+              if (stat.isFile()) {
+                return
+              }
+            } catch (e) {
+              // File not found
+            }
+          }
+
+          deleted.push({
+            _id: doc._id,
+            _rev: doc._rev,
+            _deleted: true
+          })
+        } catch (err) {
+          logger.error({ err, doc })
+          startkey = doc._id
+        }
+      }))
```

Comment:
If you page using `startkey` you might as well put `bulkDocs` in here to reduce memory usage from `deleted`.